src/deployment/deploy.py

```diff
@@ -251,6 +251,24 @@ def check_region(self) -> None:
     def create_password(self, object_id: UUID) -> Tuple[str, str]:
         return add_application_password(object_id, self.get_subscription_id())
 
+    def get_url(self) -> str:
```

Comment:
Can we unit test `get_url()` and `get_identifier_uri()`?

Reply:
Are there more specific names we should maybe choose, instead of `get_url()`? That one is really generic. It would be helpful to even just add a doc/function comment about _what_ UR{L,I}s are supposed to be returned here and below.

Reply:
I am not exactly sure what is worth testing here. The point was just to extract to a function the logic for building the url. I will update get_url to get instance_url.

Reply:
added comments

Reply:
Thank you, those comments help a lot!

In the PR description, you say "the identifier was incorrect". I was thinking we could add tests that check that the identifiers look correct, in each case. This would then naturally document examples of "correct" for both the single and multi-tenant cases.

Reply:
(Maybe the identifiers were always well-formed, but were just using the wrong sometimes, and that's why my question is confusing?)

Reply:
incorrect might not have been a good word for it. I updated the description to use mismatched instead. What is important is that the value in the request matches the value in the configuration. 
The integration test (assuming that we will have them for a multi tenant instance will catch this kind of issue)